ravenpy/models/emulators/sacsma.py

```diff
@@ -0,0 +1,672 @@
+        # Land Use Parameters
+        #-----------------------------------------------------------------
+        :LandUseParameterList
+          :Parameters,      GAMMA_SHAPE,      GAMMA_SCALE,      MELT_FACTOR,  STREAM_FRACTION, MAX_SAT_AREA_FRAC,      BF_LOSS_FRACTION
+               :Units,                -,                -,           mm/d/C,           [0..1],            [0..1],                [0..1]
+            [DEFAULT], {params.par_x18}, {params.par_x19}, {params.par_x17}, {params.par_x15},  {params.par_x14},    {derived_params.PAR_BF_LOSS_FRAC}
+        #                      para_x18,         para_x19,         para_x17,           <RIVA>,           <ADIMP>,     <SIDE>/(1+<SIDE>)
+        #                      para_x18,         para_x19,         para_x17,         para_x15,          para_x14, para_x12/(1+para_x12)
+        :EndLandUseParameterList
+
+        #-----------------------------------------------------------------
+        # Vegetation Parameters
+        #-----------------------------------------------------------------
+        :VegetationParameterList
+          :Parameters,  RAIN_ICEPT_PCT,  SNOW_ICEPT_PCT,
+               :Units,               -,               -,
+            [DEFAULT],             0.0,             0.0,
+        :EndVegetationParameterList
+        """
+
+        self.config.rvp.set_tmpl(rvp_tmpl)
+
+        #########
+        # R V I #
+        #########
+
+        rvi_tmpl = """
+        :PotentialMeltMethod     POTMELT_DEGREE_DAY
+        :RainSnowFraction        {rain_snow_fraction}
+        :Evaporation             {evaporation}
+        :CatchmentRoute          ROUTE_GAMMA_CONVOLUTION
+        :Routing                 ROUTE_NONE
+
+        :SoilModel               SOIL_MULTILAYER 7
+
+        :Alias UZ_T  SOIL[0]
+        :Alias UZ_F  SOIL[1]
+        :Alias LZ_T  SOIL[2]
+        :Alias LZ_PF SOIL[3]
+        :Alias LZ_PS SOIL[4]
+
+        :HydrologicProcesses
+          :SnowBalance           SNOBAL_SIMPLE_MELT   SNOW          PONDED_WATER
+          :Precipitation         RAVEN_DEFAULT        ATMOS_PRECIP  MULTIPLE
+          :SoilEvaporation       SOILEVAP_SACSMA      MULTIPLE      ATMOSPHERE
+          :SoilBalance           SOILBAL_SACSMA       MULTIPLE      MULTIPLE
+          :OpenWaterEvaporation  OPEN_WATER_RIPARIAN  SURFACE_WATER ATMOSPHERE
+        :EndHydrologicProcesses
+        """
+        self.config.rvi.set_tmpl(rvi_tmpl)
+
+        self.config.rvi.rain_snow_fraction = RVI.RainSnowFractionOptions.DATA
+        self.config.rvi.evaporation = RVI.EvaporationOptions.PET_OUDIN
+
+    def derived_parameters(self):
+
+        params = cast(SACSMA.Params, self.config.rvp.params)
+        self.config.rvp.derived_params = SACSMA.DerivedParams(
+            PAR_BF_LOSS_FRAC=params.par_x12,
+            POW_X01=params.par_x01,
+            POW_X02=params.par_x02,
+            POW_X03=params.par_x03,
+        )
+
+        soil0 = params.par_x04 * 1000.0
```

Comment:
Maybe it would be helpful to name them here as we call them above (i.e. `par_soil0_mm`)? Same for the one in next line.